examples/megacli.cpp

```diff
@@ -4740,6 +4740,14 @@ void DemoApp::account_updated()
     }
 }
 
+void DemoApp::notify_confirmation(const char *email)
+{
+    if (client->loggedin() == EPHEMERALACCOUNT)
+    {
+        LOG_debug << "Account has been confirmed with email " + string(email) + ". Proceed to login with credentials.";
```

Comment:
The `string()` constructor could be removed if `<<`  is used instead of `+`. That would also prevent a crash if `email` is NULL.